src/containers/Home/dashboardHome.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+import ArrowForwardIosSharpIcon from "@mui/icons-material/ArrowForwardIosSharp";
+import MuiAccordion, { AccordionProps } from "@mui/material/Accordion";
+import MuiAccordionSummary, {
+    AccordionSummaryProps,
+} from "@mui/material/AccordionSummary";
+import Typography from "@mui/material/Typography";
+import { UIConstants } from "@constants/UIConstants";
+import { AccordionDetails, DivStyle } from "./dashboardHome.style";
+
+const Accordion = styled((props: AccordionProps) => (
+    <MuiAccordion disableGutters elevation={0} square {...props} />
+))(({ theme }) => ({
+    border: `1px solid ${theme.palette.divider}`,
+    "&:not(:last-child)": {
+        borderBottom: 0,
+    },
+    "&:before": {
+        display: "none",
+    },
+}));
+
+const AccordionSummary = styled((props: AccordionSummaryProps) => (
+    <MuiAccordionSummary
+        expandIcon={<ArrowForwardIosSharpIcon sx={{ fontSize: "0.9rem" }} />}
+        {...props}
+    />
+))(({ theme }) => ({
+    backgroundColor:
+        theme.palette.mode === "dark"
+            ? "rgba(255, 255, 255, .05)"
+            : "rgba(0, 0, 0, .03)",
+    flexDirection: "row-reverse",
+    "& .MuiAccordionSummary-expandIconWrapper.Mui-expanded": {
+        transform: "rotate(90deg)",
+    },
+    "& .MuiAccordionSummary-content": {
+        marginLeft: theme.spacing(1),
+    },
+}));
+
+export default function CustomizedAccordions() {
+    const [expanded, setExpanded] = React.useState<string | false>("panel1");
+
+    const handleChange =
+        (panel: string) =>
+        (event: React.SyntheticEvent, newExpanded: boolean) => {
+            setExpanded(newExpanded ? panel : false);
+        };
+
+    return (
+        <div style={DivStyle}>
+            <Accordion
+                expanded={expanded === "panel1"}
```

Comment:
Please use the proper naming convention for the accordion name.

https://google.github.io/styleguide/tsguide.html#syntax